compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeContainer.desktop.kt

```diff
@@ -203,9 +203,11 @@ internal class ComposeContainer(
         windowContext.setWindowFocused(isFocused)
         mediator.onChangeWindowFocus()
         layers.fastForEach(DesktopComposeSceneLayer::onChangeWindowFocus)
-        lifecycle.handleLifecycleEvent(
-            event = if (isFocused) Lifecycle.Event.ON_RESUME else Lifecycle.Event.ON_PAUSE
-        )
+        if (isFocused) {
+            lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_RESUME)
+        } else if (window?.isVisible == true) {
```

Comment:
Not sure that using `isVisible` is the right here. `isVisible` doesn't mean that it is not iconified. We should store our own property, if AWT doesn't have one.